src/pages/en/persona.md

```diff
@@ -0,0 +1,603 @@
+---
+title: Persona
+description: The main goal of creating these personas was to better understand our target audience. By developing a greater understanding of our users. By developing a greater understanding of our users, we would be able to build a responsive site architecture for all existing and future content hosted on the Digital Accessibility Toolkit’s website.
+---
+
+### The creation of our persona
```

Comment:
change all the heading by -1 #.

So the page should start at `##` for the first heading